docs/yul.rst

```diff
@@ -152,7 +152,7 @@ where an object is expected.
 Inside a code block, the following elements can be used
 (see the later sections for more details):
 
-- literals, i.e. ``0x123``, ``42`` or ``"abc"`` (strings up to 32 characters)
+- literals, e.g. ``0x123``, ``42`` or ``"abc"`` (strings up to 32 characters)
```

Comment:
I think `e.g.` was actually more correct here. `i.e.` would suggest that these are all possible literals, but they're just examples.

Reply:
Ah, wait, my bad. That's exactly what you did here :) It's fine then.